mm/map/Make.defs

```diff
@@ -23,7 +23,7 @@
 CSRCS += mm_map.c
 
 ifeq ($(CONFIG_ARCH_VMA_MAPPING),y)
-CSRCS += vm_region.c
+CSRCS += vm_region.c vm_map.c
```

Comment:
why not put vm_map_region into vm_region.c?

Reply:
To make this patch focus on drivers layer, can we do that separately?

Reply:
OK

Reply:
thanks, will do it later separately.